### PR TITLE
Add tag's description

### PR DESCRIPTION
**Pull request by ganmacs**
which was introduced in https://github.com/fluent/fluent-plugin-prometheus/pull/129

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -369,7 +369,12 @@ Reserved placeholders are:
 - `${worker_id}`: fluent worker id
 - `${tag}`: tag name
   - only available in Prometheus output/filter plugin
-
+- `${tag_parts[N]}` refers to the Nth part of the tag.
+  - only available in Prometheus output/filter plugin
```
Comment (cosmo0920) [inline README.md]: :+1:

File: README.md [hunk 2]
```diff
@@ -369,7 +369,12 @@ Reserved placeholders are:
...
+- `${tag_prefix[N]}` refers to the [0..N] part of the tag.
+  - only available in Prometheus output/filter plugin
```
Comment (cosmo0920) [inline README.md]: :+1:

File: README.md [hunk 3]
```diff
@@ -369,7 +369,12 @@ Reserved placeholders are:
...
+- `${tag_suffix[N]}` refers to the [N..] part of the tag.
```
Comment (cosmo0920) [inline README.md]: Is tag_suffix[N] behavior correctly described?
I'm confusing this placeholder behavior:

Where _tags_ are `1.2.3`:

`${tag_suffix[0]}` is expanded to `3`.  But the description insists [0..], so the placeholder will be `1.2.3`.
`${tag_suffix[1]}` is expended to `2.3`. But the description insists [1..], so the placeholder will be `2.3`.
`${tag_suffix[2]}` is expanded to `1.2.3`. But the description insists [2..], so the placeholder will be `3`.

Am I wrong? :confounded:

Comment (ganmacs) [reply README.md]: Good catch... https://github.com/fluent/fluent-plugin-prometheus/pull/137/commits/123979c96653b88b04b3443b7ebbfd98b6da3856#diff-04c6e90faac2675aa89e2176d2eec7d8R376

https://github.com/fluent/fluent-plugin-prometheus/blob/790a87d66bee29b945e5e6f8425b7c8fe2ce09f7/spec/fluent/plugin/prometheus/placeholder_expander_spec.rb#L41

Comment (cosmo0920) [reply README.md]: Gotcha! Thanks for the quick fixing! :wink:

**Comment by ganmacs** [comment]
thanks!

**Pull request merged**